src/components/App.tsx

```diff
@@ -80,20 +80,22 @@ class App extends React.Component<IAppProps, IAppState> {
         }
     }
 
-    private redirectToLogin = async () => {
+    private init = async () => {
```

Comment:
Overall I find that the logic could be improved. It feels like a race condition that we call async method init, without awaiting it. What happens if init call takes longer than what it takes on our development machines. If init takes longer, then application will load as usually, but there will be errors, since we are not logged in. Then finally when the init() method completes, it will redirect the user to login page. I think this should be made so that the page is viewed right from the start, but App.tsx should only show a loading icon, as long as init() function is loading. When init function is done, we can then let the user access the app, or redirect the user to login page.

Reply:
I know that your PR do not have anything to do with this thing, we could probably create an issue to fix this.